specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/LogicAppsExtension.json

```diff
@@ -0,0 +1,697 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-02-02-preview",
+    "title": "ContainerApps API Client",
+    "description": "Logic app is an extension resource on container app and the extension resource enables logic specific operations on the container app through the APIs listed."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
Would you please share more details about why modeling the logicApp resource as an extension scoped resource? And it seems only applied to containerApp resource?

How about modeling the logicApp resource as a nested resource of containerApp?

Reply:
So the logic app extension resource adds logic apps specific capapabilities on the container app. So it made more sense as an extension resource rather than keeping it as a nested resource

Reply:
Thanks for the clarification, can this logic app extension apply to another resource type?

Reply:
No only the container app resource for now